tests/yam/yast_modules/ntp_configuration.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module launches the installation from
+#          the installation settings page.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use Test::Assert 'assert_matches';
+
+sub run {
+    my $ntp_settings = $testapi::distri->get_ntp_configuration()->get_ntp_value();
+    assert_matches(qr/suse\.pool\.ntp\.org|10\.160\.0\.45\ 10\.160\.0\.44\ 10\.160\.255\.254/, $ntp_settings, "NTP setting is not correct");
```

Comment:
in general we don't do this, this is a weak validation, if the servers where swap we would not detect any issue, better practice is not validate the screen in 99 % of the test suite and then have a extended test suite that does all the heavy validation via test data (without any condition, 'OR" or anything. But you can keep it like that, just pointing you to the better solution for future.